libs/barista-components/micro-chart/src/micro-chart.ts

```diff
@@ -367,42 +378,51 @@ function checkUnsupportedType(type?: string): void {
   }
 }
 
-function checkUnsupportedOptions(options: DtChartOptions): void {
+function checkUnsupportedOptions(options: DtMicroChartOptions): void {
   if (options.chart) {
     checkUnsupportedType(options.chart.type);
   }
 }
 
 /** Apply count formatter to value to be displayed in data label */
 function addDataLabelFormatter(
-  dataPoint: DataPoint,
+  dataPoint: SeriesColumnDataOptions | SeriesLineDataOptions,
   formatter: (input: number) => string,
 ): void {
+  // TODO remove code duplication
```

Comment:
Is there an issue for this TODO?

If not please create one and add the `quality` label on it and maybe link it in the comment.

Reply:
TODO removed, comment added. Thanks.